minilib/src/main/scala/java/lang/Class.scala

```diff
@@ -0,0 +1,19 @@
+package java.lang
+
+import scala.scalajs.js
+
+final class Class[A] private (private val data: Object) extends Object {
```

Comment:
I think this class (and all classes in `minilib`) deserve a comment why they are here (and not just pulled from `javalib`).

For this class, I guess it is: "Stripped-down version of jl.Class that only provides what the linker needs itself.".

Reply:
Yes, indeed. And in this case, the full one wants a static type for its `data`. If we wanted to keep the full one, we would have needed to drop the static type of `data` in the actual lib, which was a pity.